src/components/structures/MatrixChat.js

```diff
@@ -1419,6 +1422,42 @@ export default React.createClass({
         }
     },
 
+    _handleSyncError(e) {
```

Comment:
Leftover? Seems identical to `Lifecycle.handleInvalidStoreError` and not used AFAICT